Makefile

```diff
@@ -58,6 +58,11 @@ docker-push:
 	@docker tag "${DOCKER_IMAGE_NAME}" improbable/"$(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG)"
 	@docker push improbable/"$(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG)"
 
+docker-tag-latest:
+	@echo ">> tag latest"
+	@docker tag improbable/"$(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG)" improbable/"$(DOCKER_IMAGE_NAME):latest"
+	@docker push improbable/"$(DOCKER_IMAGE_NAME):latest"
```

Comment:
Can we remove pushing the `latest`? 
As I already said in previous comment:

>  Anyway, I think we want to get rid of latest totally, because it is super confusing when users claim answers (when asked what Thanos version you use) just "latest" ;p Can we just publish vX.X.X?

Reply:
I suppose, `latest` is a standard docker convention. But if you don't want to support it, that's fine.